packages/block-library/src/cover/index.php

```diff
@@ -18,56 +18,48 @@ function render_block_core_cover( $attributes, $content ) {
 		return $content;
 	}
 
-	$current_featured_image = get_the_post_thumbnail_url();
-
-	if ( false === $current_featured_image ) {
+	$is_image_background = 'image' === $attributes['backgroundType'];
+	if ( ! $is_image_background ) {
 		return $content;
 	}
 
-	$is_img_element      = ! ( $attributes['hasParallax'] || $attributes['isRepeated'] );
-	$is_image_background = 'image' === $attributes['backgroundType'];
-
-	if ( $is_image_background && ! $is_img_element ) {
-		$content = preg_replace(
-			'/class=\".*?\"/',
-			'${0} style="background-image:url(' . esc_url( $current_featured_image ) . ')"',
-			$content,
-			1
-		);
-	}
+	$is_img_element = ! ( $attributes['hasParallax'] || $attributes['isRepeated'] );
 
-	if ( $is_image_background && $is_img_element ) {
+	if ( $is_img_element ) {
 		$object_position = '';
 		if ( isset( $attributes['focalPoint'] ) ) {
-			$object_position = round( $attributes['focalPoint']['x'] * 100 ) . '%' . ' ' .
-			round( $attributes['focalPoint']['y'] * 100 ) . '%';
+			$object_position = round( $attributes['focalPoint']['x'] * 100 ) . '%' . ' ' . round( $attributes['focalPoint']['y'] * 100 ) . '%';
 		}
 
-		$image_template = '<img
-			class="wp-block-cover__image-background"
-			alt="%s"
-			src="%s"
-			style="object-position: %s"
-			data-object-fit="cover"
-			data-object-position="%s"
-		/>';
-
-		$image = sprintf(
-			$image_template,
-			esc_attr( get_the_post_thumbnail_caption() ),
```

Comment:
This was in fact incorrect and could result in accessibility problems, since it was using the caption as the `alt` attribute, when in fact WordPress images have their own dedicated field to define the alternative text.

So fixing that is another benefit of this PR.